packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -444,11 +472,62 @@ export default class QuanticCategoryFacet extends LightningElement {
     return path.join('/');
   }
 
-  setFocusToFirstFacetValue() {
+  /**
+   * Sets the focus on the target element.
+   */
+  setFocusOnTarget() {
+    this.focusShouldBeInFacet = false;
+    if (!this.focusTarget) {
+      return;
+    }
+    // eslint-disable-next-line default-case
+    switch (this.focusTarget.type) {
+      case 'facetHeader':
+        this.setFocusOnHeader();
+        return;
+      case 'facetValue':
+        if (this.values.length) {
+          this.setFocusOnFirstFacetValue();
+        } else {
+          this.setFocusOnLastNonActiveParent();
+        }
+    }
+  }
+
+  /**
+   * Sets the focus on the first facet value.
+   */
+  setFocusOnFirstFacetValue() {
     const focusTarget = this.template.querySelector(
-      'c-quantic-category-facet-value'
+      '.facet__value > c-quantic-category-facet-value'
+    );
+
+    if (focusTarget) {
+      // @ts-ignore
```

Comment:
I think you should be able to export a type in quantic-facet-value right? and set the type annotation here?